### PR TITLE
Feature/cancel draft part 2 rebased

### PR DESCRIPTION
**Pull request by rej156**

### CODE REVIEW

File: packages/kauri-components/components/Layout/Masonry.tsx [hunk 1]
```diff
@@ -45,24 +45,26 @@ class Masonry extends Component<IProps, IState> {
...
 
-  componentDidMount () {
-    window.addEventListener('resize', this.updateColumns);
+  public componentDidMount() {
+    window.addEventListener("resize", this.updateColumns);
   }
 
-  componentWillUnmount () {
-    window.removeEventListener('resize', this.updateColumns);
+  public componentWillUnmount() {
```
Comment (davodesign84) [inline packages/kauri-components/components/Layout/Masonry.tsx]: Do we need `public`? that's the default

File: packages/kauri-components/components/Layout/Masonry.tsx [hunk 2]
```diff
@@ -45,24 +45,26 @@ class Masonry extends Component<IProps, IState> {
...
+    window.removeEventListener("resize", this.updateColumns);
   }
 
-  render () {
+  public render() {
     const { children } = this.props;
-    const columnsArray: React.ReactElement<any>[][] = [];
+    const columnsArray: Array<Array<React.ReactElement<any>>> = [];
```
Comment (davodesign84) [inline packages/kauri-components/components/Layout/Masonry.tsx]: I think I prefer the other syntax

File: packages/kauri-components/components/Tabs/index.tsx [hunk 3]
```diff
@@ -20,87 +20,89 @@ interface ITabsProps {
...
-            <Tab key={index}
+      <TabContainer minWidth={minWidth}>
+        <Tabs bg={bg} padContent={padContent} centerTabs={centerTabs}>
+          {this.props.tabs.map((tab, index) => (
+            <Tab
+              key={index}
               minWidth={minWidth}
               selected={index === this.state.selectedTabIndex}
-              onClick={() => this.changeTab(index)}
+              onClick={handleClick(index)}
```
Comment (davodesign84) [inline packages/kauri-components/components/Tabs/index.tsx]: Wouldn't this execute on render?

File: packages/kauri-components/components/Card/ArticleCard.tsx [hunk 4]
```diff
@@ -309,85 +383,70 @@ const ArticleCard = ({
...
+  isLoggedIn,
+  hoverChildren,
+}) => {
+  const [{ toggledOn }, dispatch] = React.useReducer<
+    IToggleState,
+    IToggleAction
+  >(toggleReducer, toggleInitialState);
+
+  return (
+    <BaseCard
```
Comment (davodesign84) [inline packages/kauri-components/components/Card/ArticleCard.tsx]: Shall we remove the hover transform on cards?

File: packages/kauri-components/components/Layout/Masonry.tsx (hunk 2 above)
Comment (rej156) [reply packages/kauri-components/components/Layout/Masonry.tsx]: Something musta happened to my prettier, I didn't change it and the method access modifiers too lol

File: packages/kauri-components/components/Tabs/index.tsx (hunk 3 above)
Comment (rej156) [reply packages/kauri-components/components/Tabs/index.tsx]: Nah the functions returns a lambda
We should prob get rid of that tslint rule that disallows lambdas in event handlers

File: packages/kauri-components/components/Card/ArticleCard.tsx (hunk 4 above)
Comment (rej156) [reply packages/kauri-components/components/Card/ArticleCard.tsx]: Sure

File: packages/kauri-components/components/Layout/Masonry.tsx (hunk 1 above)
Comment (rej156) [reply packages/kauri-components/components/Layout/Masonry.tsx]: Yeah just checked, tslint complains

**Pull request merged**